simple_ot2_opentrons_flow.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+"""
+Minimal Prefect + Opentrons Integration Example
+Simple proof of concept showing both APIs working in same script
+"""
+
+import os
+import sys
+
+# Configure Prefect using environment variables (no hardcoded credentials)
+if 'PREFECT_API_URL' not in os.environ:
```

Comment:
Prefect is the one who looks for this by default, correct? If so, no need to reply. If not, please clarify